### PR TITLE
Fixup the BaselinePingTest

### PR DESCRIPTION
**Pull request by Dexterp37**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by Dexterp37** [comment]
This currently blocks #10146

File: app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt [hunk 1]
```diff
@@ -58,6 +59,7 @@ class BaselinePingTest {
 
     private fun waitForPingContent(
         pingName: String,
+        pingReason: String?,
```
Comment (badboy) [inline app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt:62]: Should we ... briefly document this function? for the sake of our future selves.

File: app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt [hunk 2]
```diff
@@ -87,9 +98,15 @@ class BaselinePingTest {
         // Move it to background.
         device.pressHome()
 
+        // Due to bug 1632184, we need move the activity to foreground and then to background
```
Comment (badboy) [inline app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt:101]: As I read this this only puts it in foreground again.
So this sentence is slightly wrong or at least not matching the code below.

**Comment by Dexterp37** [comment]
@isabelrios this is now failing, but the result [does not point at glean](https://console.firebase.google.com/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/4798615227619028309/executions/bs.457ca5df221c25f4). Is this correct?

**Pull request merged**